torch/onnx/_internal/fx/serialization.py

```diff
@@ -80,7 +80,7 @@ def save_model_with_external_data(
     model_location: str,
     initializer_location: str,
     torch_load_paths: Tuple[str, ...],
-    onnx_model: onnx.ModelProto,
+    onnx_model: "onnx.ModelProto",
```

Comment:
nit: this is not needed with `from __future__ import annotations`? (which is already there)